.github/workflows/draft_release.yml

```diff
@@ -0,0 +1,72 @@
+name: Draft Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Release date YYYY.MM.DD Also used to generate the tag name.'
+        required: true
+
+jobs:
+  draft_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create Release Draft
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        shell: bash
+        run: |
+          # Format current date as YYYY.MM.DD
+
+          if [[ "${{ github.event_name}}" == 'workflow_dispatch' ]]; then
+            tag="${{ github.event.inputs.tag }}"
+          else
+            tag=$(date +'%Y.%m.%d')
```

Comment:
Comment this out and add an error. this should only be used for testing PRs.

Reply:
I don't understand what you mean here - why would PRs be creating releases?

Reply:
IF you need to work on this yml file, you might want to see what release would be produced. I think adding that code into master is a bad idea though as it lead to the incorrect date on the release doc you were asking about earlier.